.gitignore

```diff
@@ -4,3 +4,5 @@
 .Ruserdata
 ggeasy.Rproj
 inst/doc
+NAMESPACE
+labs_utils.R
```

Comment:
Accidentally ignored?

Reply:
`lab_utils.R` is the +gg variation of `easy_labs`...i'll keep it for bookkeeping